File: B/cpp/code.cpp
#include <iostream>

int main() {
  int count, el, result{0}, max{0};
  scanf("%d", &count);
  while (count--) {
    scanf("%d", &el);
    if (el == 1) {
      result++;
    } else {
      max = std::max(max, result);
      result = 0;
    }
  }
  printf("%d\n", std::max(max, result));
  return 0;
}

File: B/python/code.py
import sys

count = sys.stdin.readline().strip()

result = 0
max = 0
for i in range(int(count)):
    el = sys.stdin.readline().strip()
    if el == '1':
        result += 1
    else:
        if max < result:
            max = result
        result = 0

print(max if max > result else result)